app-component.html

```diff
@@ -35,13 +35,11 @@ <h2 class="headline">Introduction</h2>
       </div>
     </div>
 
-    <hr ng-if="::!$ctrl.config.productionMode" />
+    <hr ng-if="::!$ctrl.config.productionMode">
     <br-demo-warning></br-demo-warning>
 
-    <div ng-if="$ctrl.config.footer.show">
-      <hr/>
-      <br-footer ng-if="$ctrl.config.footer.show"></br-footer>
-    </div>
+    <hr ng-if="!$ctrl.config.footer._hideOuterHr">
+    <br-footer ng-if="$ctrl.config.footer.show"></br-footer>
```

Comment:
Holding off on this for now until there is a better design that covers footer, navbar, warnings, etc.